### DISCUSSION TITLE
How to dynamically control VCardTitle background color in Trame/Vuetify3?

### DISCUSSION THREAD

**Discussion by Roadelse** [Kitware/trame#642] [Q&A] [answered]
I'm trying to set a dynamic background color for a vuetify3.VCardTitle component. Since VCardTitle doesn't have a built-in background_color property (as shown in [trame doc](https://trame.readthedocs.io/en/latest/trame.widgets.vuetify3.html#trame.widgets.vuetify3.VCardTitle) and [vuetify3 doc](https://vuetifyjs.com/en/api/v-card-title/)), how can I bind a variable to its background color?

I'm not familiar with vue and have tried several approaches (some from AI):

+ `style="color: {{ bkg_color }}"`
+ `style="{{ my_style }}"`
+ `__properties=[("v-bind:style", "my_style")]`

but none of these worked

Then I find this example: [28_css](https://github.com/Kitware/trame/blob/master/examples/validation/core/28_css.py), but it only shows how to change global CSS styles rather than targeting a specific element.

Thus, I wonder what's the proper, or recommended way to bind a style variable to a specific VCardTitle component?

Thank you in advance for any answer!

**Comment by jourdain** [comment, marked as answer]
```
style=("{ color: bkg_color }",),
```
or
```
style=("`color: ${bkg_color};`",),
```

**Reply by Roadelse** [reply]
Thanks a lot! Looks like I've missed out on quite a bit of Vue knowledge :(

**Comment by jourdain** [comment]
The usage of `{{ var }}` is for text template not properties. 

Also if you want to use `__properties=` you should have your tuple defined as `(py_name, js_attr)`.
```
my_style="`color: ${bkg_color};`",
__properties=[(my_style, ":style")],
```
